src/kimchima/pkg/cross_encoder_factory.py

```diff
@@ -0,0 +1,282 @@
+from transformers import AutoModelForSequenceClassification, AutoTokenizer, AutoConfig
```

Comment:
I'd like to add a linter to check the styles of the  import libs. However, it should be not an issue in this PR.

Reply:
I want to mentioned, if we use the code from other repos. We should add

"""
Original code from: URL
Adapted by: Your name
Date: adopt date
"""

We acknowledge their work, and hope they do same to others

Reply:
Add comments for original code